hooks/useError.tsx

```diff
@@ -4,7 +4,26 @@ import { INSERT_ERROR } from '../services/GraphQL/mutations/errors'
 const useError = () => {
   const [createError] = useMutation(INSERT_ERROR)
 
-  return [createError]
+  const logError = (
+    error: Error,
+    codeLocation: string,
+    type: 'INPUT' | 'AUTHORIZATION' | 'AVAILABILITY' | 'UNEXPECTED' | 'OTHER' = 'OTHER',
+  ): void => {
+    if (process.env.NODE_ENV !== 'production') {
+      console.error(codeLocation, error)
+    } else {
+      createError({
+        variables: {
+          error: JSON.stringify(error),
+          origin: 'ADMIN',
+          type,
+          codeLocation,
+        },
+      })
+    }
+  }
+
+  return [logError]
```

Comment:
Was thinking on add more methods related to log, debug and show notifications/toasts. Can you guys help me to think about it?

Reply:
I think `logError` could return a `show` property, and if this is `true` shows a toast

Reply:
What about something like "showNotification" or similar?

Reply:
I updated the methods, please check again